test/runnable/no_assertz_cov.d

```diff
@@ -0,0 +1,11 @@
+/*
+POST_SCRIPT: runnable/extra-files/no_assertz_cov-postscript.sh
+REQUIRED_ARGS: -cov -main
+*/
+module no_assertz_cov;
+
+void nocov()
+{
+    assert(0);
+}
```

Comment:
```suggestion
void nocov()
{
    int a = 42;
    if (!a)
        assert(0);
}
```

Then you can use `-cov=100` and don't need the extra file and the script.

Reply:
so much better, thanks.